scripts/jenkins/cloud/ansible/roles/heat-generator/templates/heat-template.yaml

```diff
@@ -23,6 +23,7 @@ parameters:
     description: Default CIDR value to use for networks which do not have CIDR specified
     default: {{ ipv6 | ternary("2001::/64", "1.1.1.0/24") }}
 
+
```

Comment:
@spacefito, in OpenStack, every interface needs to have an associated network and subnet (i.e.  you cannot create a dangling port). This default network/subnet is used for input model interfaces that only have associated VLAN networks, but no native network. It really doesn't matter what you use in that case, because the native interface should never be used. This is just a dummy network/subnet.